src/test/java/io/confluent/connect/jdbc/sink/JdbcSinkTaskTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ *  Copyright 2016 Confluent Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.confluent.connect.jdbc.sink;
+
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.errors.RetriableException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.kafka.connect.sink.SinkTaskContext;
+import org.easymock.EasyMockSupport;
+import org.junit.Test;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static org.easymock.EasyMock.expectLastCall;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class JdbcSinkTaskTest extends EasyMockSupport {
+
+  @Test
+  public void retries() {
+    final int maxRetries = 2;
+    final int retryBackoffMs = 1000;
+
+    Set<SinkRecord> records = Collections.singleton(new SinkRecord("stub", 0, null, null, null, null, 0));
+    final JdbcDbWriter mockWriter = createMock(JdbcDbWriter.class);
+    SinkTaskContext ctx = createMock(SinkTaskContext.class);
+
+    try {
+      mockWriter.write(records);
```

Comment:
Doesn't much matter in this case since there's only one call, but I've also found that for these tests, adding a `throws SQLException` annotation to the test works fine in place of actually catching it and rethrowing since we'd never be throwing `SQLExceptions` ourselves anyway.